process/transaction/metaProcess.go

```diff
@@ -145,6 +145,8 @@ func (txProc *metaTxProcessor) ProcessTransaction(tx *transaction.Transaction) (
 		if txProc.enableEpochsHandler.IsFlagEnabled(common.ESDTFlag) {
 			return txProc.processSCInvoking(tx, tx.SndAddr, tx.RcvAddr)
 		}
+	case process.RelayedTxV3:
```

Comment:
this is not the best solution.

SCR does not need to be created. You have all the data on the transaction.

Reply:
updated the execution to follow the normal tx flow